docs/deployment/environment-variables.md

```diff
@@ -86,6 +86,7 @@ More information about distributed deployment are available [here](distributed.m
 | LOG_LEVEL                               | info                       | v0.6   | —      | one of trace, debug, info, warn or error                                                       |
 | JAVA_TOOL_OPTIONS                       | —                          | —      | —      | JVM options \(Docker only\)                                                                    |
 | FHIR_OPERATION_EVALUATE_MEASURE_THREADS | 4                          | v0.8   | —      | The maximum number of parallel $evaluate-measure executions.                                   |
+| FHIR_OPERATION_EVALUATE_MEASURE_TIMEOUT | 3600000 (1h)               | v0.19  | —      | Timeout in milliseconds for $evaluate-measure executions.                                      |
```

Comment:
```suggestion
| FHIR_OPERATION_EVALUATE_MEASURE_TIMEOUT | 3600000 (1h)               | v0.19  | 3600000 | Timeout in milliseconds for $evaluate-measure executions.                                      |
```

Reply:
As noted in the PR description

Reply:
That column is for the version of deprecation. The default value is already there.